css-view-transitions-2/Overview.bs

```diff
@@ -523,6 +520,8 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 		1. Set |oldDocument|'s [=active view transition=] to |outboundTransition|.
 
 			Note: The process continues in [=setup view transition=], via [=perform pending transition operations=].
+
+		1. The user agent should display the currently displayed frame until |newDocument| is [=revealed=], or until its [=active view transition=]'s [=ViewTransition/phase=] is "`done`".
```

Comment:
Can you add a non-normative note here clarifying why this is needed.

Reply:
Done